app/src/main/java/com/bc/gordiansigner/ui/sign/PsbtSignViewModel.kt

```diff
@@ -25,7 +26,8 @@ class PsbtSignViewModel(
     private val rxLiveDataTransformer: RxLiveDataTransformer
 ) : BaseViewModel(lifecycle) {
 
-    internal val psbtSigningLiveData = CompositeLiveData<Pair<String, KeyInfo?>>()
+    internal val psbtSigningLiveData = CompositeLiveData<String>()
+    internal val getKeyToSignLiveData = CompositeLiveData<KeyInfo>()
```

Comment:
Notice that you should define all `LiveData` variables name in consistent way. For example, if the one is `psbtSigningLiveData`, the other one should be `keyToSignGettingLiveData`. The easiest way is to name them is follow the corresponding method name, e.g.,`checkPsbtLiveData` or `getKeyToSignLiveData`.